src/app/common/Placeholder.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from 'terra-image';
+
+const propTypes = {
+  /**
+   * The image source to display.
+   */
+  src: PropTypes.string,
+};
+
+const defaultProps = {
+  src: undefined,
+};
+
+const Placeholder = ({ src }) => (
```

Comment:
Future note, we might want to export this component for people to use in their doc, it makes for nice examples.